examples/next-prisma-websockets-starter/src/server/routers/post.ts

```diff
@@ -46,7 +46,9 @@ const interval = setInterval(() => {
     ee.emit('isTypingUpdate');
   }
 }, 3e3);
-process.on('SIGTERM', () => clearInterval(interval));
+process.on('SIGTERM', () => {
+  clearInterval(interval);
+});
```

Comment:
https://v6--typescript-eslint.netlify.app/rules/no-confusing-void-expression flags cases around `void` such this: a function that returns a value being used in a place that expects it to return `void`. It's in the _`stylistic-type-checked`_ config in v6, after not being in any recommended configs in v5. I worry it's too opinionated and will annoy people. Thoughts?

Reply:
I think it's a good rule